src/org/traccar/protocol/MiniFinderProtocolDecoder.java

```diff
@@ -34,44 +34,145 @@ public MiniFinderProtocolDecoder(MiniFinderProtocol protocol) {
         super(protocol);
     }
 
-    private static final Pattern PATTERN = new PatternBuilder()
-            .expression("![A-D],")
+    // The !A record is not in the documentation for MiniFinder or EV-07S.
+    // Eview, the manufacturer, indicated that resellers could implement new record types.
+    // Using the data in MiniFinderProtocolDecoderTest.java as a guide
+    private static final Pattern PATTERN_A = new PatternBuilder()
+            .expression("!A,")
             .number("(d+)/(d+)/(d+),")           // date
```

Comment:
I think we should remove duplication in the patter. See `Gl200ProtocolDecoder.PATTERN_LOCATION` for example.